src/app/App.js

```diff
@@ -58,27 +59,15 @@ export const App = ({ options }) => {
     }),
     inside: mainContainer,
   });
-
-  const odpTrue = render({
-    component: Button({
-      id: 'odpTrue',
-      btnText: 'odpTrue',
-      classList: ['mainContainer__odpTrue'],
-      onClickFn: startGame,
-      isCorrectAnswer: true,
-    }),
-    inside: mainContainer,
-  });
-
-  const odpFalse = render({
-    component: Button({
-      id: 'odpFalse',
-      btnText: 'odpFalse',
-      classList: ['mainContainer__odpFalse'],
-      onClickFn: '',
-      isIncorrectAnswer: true,
-    }),
+  const answers = render({
+    component: AnswersWrapper(
+      ['Luke Skywalker', 'Jar Jar Binks', 'Padme Amidala', 'Darth Vader'],
+      'Darth Vader',
+      chosenAnswer,
+    ),
+    //component: AnswerButtons(answers, correctAnswer, chosenAnswer),
```

Comment:
1) AnswersWrapper chyba nie może wiedzieć jaka jest poprawna odpowiedź!
2) Po kliknięciu i nie puszczeniu widać która jest poprawna

Reply:
https://user-images.githubusercontent.com/41823689/104111363-4396af00-52e1-11eb-9f5d-d66671daad7d.mp4

Reply:
Zobacz sobie PR "Fixed Btn"
To powinno Ciebie zadowolić

Reply:
Zobacz sobie PR "Fixed Btn"
To powinno Ciebie zadowolić

Reply:
Chyba OK, było w tasku, żeby zaznaczało czy odpowiedziałeś poprawnie, czy źle. 
Bo jak klikniemy to i tak chyba nie możemy się wycofać nie (chyba że jakoś możemy) :) ? Nie ważne, że jeszcze nie puściliśmy.

Reply:
To wina "focus" w scss

Reply:
1.  Komponent przyjmuje na wejściu odpowiedzi na pytanie.
Komponent przyjmuje na wejściu oznaczenie, która odpowiedź jest prawidłowa.
Tylko nazwa faktycznie jednak słaba.
2. :visited będzie ok?

Reply:
@nowakprojects No właśnie możemy - bo to :focus.
Czyli nie jest liczony jako onClick, tylko jako "tab" (trochę takie najechanie kursorem)
(dodatkowo Na telefonach odpowiedzi zaznaczałyby się podczas scrolowania)

@lamparina 
Ok, komponent, ale nie przyciski!
Przyciski nie muszą, a może nawet nie powinny wiedzieć jaka jest odpowiedć.
Komponent wrappera tak, ale przyciski nie.
One powinny dowiedzieć się dopiero po sprawdzeniu poprawności

Reply:
@PiotrWR @lamparina 
Ad 2.
Jak dla mnie ciągle jest problem, z tym że sądzimy, że to przycisk. A to jest odpowiedź na pytanie. To czy wygląda jak przycisk, czy jest tekstem, skaczącą myszką. Nie ma znaczenia :) Ja nawet bym się nie poskarżył jakby nie używała w ogóle buttona a cały kod byłby powtórzony (ale na razie nie widzę takiej potrzeby). Ten komponent jeden odpowiedzi raczej musi gdzieś tam kiedyś się dowiedzieć, żeby zaznaczyć się na czerwono / zielono po kliknięciu.